go/service/pgp.go

```diff
@@ -162,7 +162,7 @@ func (h *PGPHandler) PGPImport(_ context.Context, arg keybase1.PGPImportArg) err
 		LogUI:     h.getLogUI(arg.SessionID),
 		SessionID: arg.SessionID,
 	}
-	eng, err := engine.NewPGPKeyImportEngineFromBytes(arg.Key, arg.PushSecret, nil)
+	eng, err := engine.NewPGPKeyImportEngineFromBytes(arg.Key, arg.PushSecret, h.G())
```

Comment:
OK, I looked into it.  I think there are two bugs here.  One, as you found, and the other that `StringWarning#Warn` should take a `libkb.Contextified` which is resilient to this type of `nil` bug.